<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setWin</class>
 <widget class="QDialog" name="setWin">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>221</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>221</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <property name="windowIcon">
   <iconset resource="source.qrc">
    <normaloff>:/图标/icon.ico</normaloff>:/图标/icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="normalTab">
      <attribute name="title">
       <string>常规</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="4">
          <widget class="QCheckBox" name="warningWinCheckBox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>倒计时小于</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>启动时预警弹窗</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="6">
          <widget class="QComboBox" name="sortRuleComboBox">
           <item>
            <property name="text">
             <string>重要性优先</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>剩余时间优先</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>小时</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>排序规则</string>
           </property>
          </widget>
         </item>
         <item row="3" column="5" colspan="2">
          <widget class="QPushButton" name="warningWinPushButton">
           <property name="text">
            <string>弹窗预览</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>天</string>
           </property>
          </widget>
         </item>
         <item row="2" column="6">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>分预警</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="countDownDaySpinBox">
           <property name="maximum">
            <number>365</number>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QSpinBox" name="countDownHourSpinBox">
           <property name="maximum">
            <number>23</number>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <widget class="QSpinBox" name="countDownMinuteSpinBox">
           <property name="maximum">
            <number>59</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>setWin</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>setWin</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>warningWinPushButton</sender>
   <signal>clicked()</signal>
   <receiver>setWin</receiver>
   <slot>showWarnWinButtonPressAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showWarnWinButtonPressAction()</slot>
 </slots>
</ui>
